docs/source/installation.rst

```diff
@@ -0,0 +1,90 @@
+.. _install:
+
+Install
+=======
+
+pygeoapi is easy to install on numerous environments.  Whether you are a user, administrator or developer, below
+are multiple approaches to getting pygeoapi up and running depending on your requirements.
+
+Requirements and dependencies
+-----------------------------
+
+pygeoapi runs on Python 3.
+
+Core dependencies are included as part of a given pygeoapi installation procedure.  More specific requirements
+details are described below depending on the platform.
+
+
+For developers and the truly impatient
+--------------------------------------
+
+.. code-block:: bash
+
+   python -m venv pygeoapi
+   cd pygeoapi
+   . bin/activate
+   git clone https://github.com/geopython/pygeoapi.git
+   cd pygeoapi
+   pip install -r requirements.txt
+   python setup.py install
+   cp pygeoapi-config.yml example-config.yml
+   vi example-config.yml
+   export PYGEOAPI_CONFIG=example-config.yml
+   export PYGEOAPI_OPENAPI=example-openapi.yml
```

Comment:
`example-openapi.yml` is not provided which will cause `pygeoapi serve` to fail. In order to get pygeoapi running, we should create at least an empty file : `echo > " " > example-openapi.yml`